test/test_1pbasis-equiv.jl

```diff
@@ -0,0 +1,50 @@
+
+# --------------------------------------------------------------------------
+# ACE.jl: Julia implementation of the Atomic Cluster Expansion
+# Copyright (c) 2019 Christoph Ortner <christophortner0@gmail.com>
+# All rights reserved.
+# --------------------------------------------------------------------------
+
+
+#@testset "1-Particle Basis"  begin
+
```

Comment:
It loos to me these tests can be deleted entirely? Or this it have a purpose I don't see yet?
If yes, then it should go into the `test_euclvec.jl` testset I think.

Reply:
can be deleted. Sorry for the clutter